src/agent/firebase-controller.ts

```diff
@@ -133,10 +133,11 @@ export class FirebaseController implements Controller {
 
     // Calculate the debuggee id as the hash of the object.
     // This MUST be consistent across all debuggee instances.
-    this.debuggeeId = crypto
+    const debuggeeHash = crypto
       .createHash('md5')
```

Comment:
Just realized we're using MD5 here, but SHA1 in the Python and Java agents.

Reply:
Good catch.  I'll switch to SHA1.  MD5 isn't FIPS compliant so it's possible it won't be present.